service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/AmendedOrderEmailContentProvider.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.config.HmctsCourtLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.model.notify.OrderAmendedNotifyData;
+import uk.gov.hmcts.reform.fpl.service.email.content.base.AbstractEmailContentProvider;
+import uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper;
+
+import static uk.gov.hmcts.reform.fpl.enums.TabUrlAnchor.ORDERS;
+import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.buildCallout;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor(onConstructor_ = {@Autowired})
+public class AmendedOrderEmailContentProvider extends AbstractEmailContentProvider {
+    private final HmctsCourtLookupConfiguration config;
+    private final EmailNotificationHelper helper;
+
+    public OrderAmendedNotifyData getNotifyData(final CaseData caseData,
+                                                final DocumentReference orderDocument,
+                                                final String orderType) {
+
+        return OrderAmendedNotifyData.builder()
+            .lastName(helper.getSubjectLineLastName(caseData))
```

Comment:
vv minor I would just use the childLastName, as the toggle is defaulted to true so you are just causing more techdebt

Reply:
actually just realised it's only respondent they want in subject, so calling diff method here now

Reply:
checked with Gail, we want child last name..